src/stateFromElement.js

```diff
@@ -78,17 +79,15 @@ const ELEM_ATTR_MAP = {
 
 const getEntityData = (tagName: string, element: DOMElement) => {
   const data = {};
-  if (ELEM_ATTR_MAP.hasOwnProperty(tagName)) {
-    const attrMap = ELEM_ATTR_MAP[tagName];
-    for (let i = 0; i < element.attributes.length; i++) {
-      const {name, value} = element.attributes[i];
-      if (value != null) {
-        if (attrMap.hasOwnProperty(name)) {
-          const newName = attrMap[name];
-          data[newName] = value;
-        } else if (DATA_ATTRIBUTE.test(name)) {
-          data[name] = value;
-        }
+  const attrMap = ELEM_ATTR_MAP[tagName];
+  for (let i = 0; i < element.attributes.length; i++) {
+    const {name, value} = element.attributes[i];
+    if (value != null) {
+      const newName = attrMap && attrMap.hasOwnProperty(name) && attrMap[name];
+      if (newName) {
+        data[newName] = value;
+      } else if (DATA_ATTRIBUTE.test(name)) {
```

Comment:
This is a totally reasonable change that's not addressed in #24. Let's maybe call this `WHITELIST_ATTRIBUTES` and make it configurable in the options. As it stands now this would be a breaking change.